python/emitter.js

```diff
@@ -114,15 +102,49 @@
         addHook('deps', 'from applitools.common.ultrafastgrid import DesktopBrowserInfo, IosDeviceInfo, ChromeEmulationInfo, ScreenOrientation')
         addHook('beforeEach', python`    conf.add_browser(${{value: test.config.browsersInfo, type: 'BrowsersInfo'}})`)
     }
+    if ("batch" in test.config) {
+        if ("id" in test.config.batch) {
+            addHook('deps', 'from datetime import datetime')
+            addHook('beforeEach', python`    conf.batch.id = ${test.config.batch.id}`)
+        }
+        if ("properties" in test.config.batch) {
+            addHook('beforeEach', python`    conf.batch.add_property(${test.config.batch.properties[0].name}, ${test.config.batch.properties[0].value})`)
+        }
+    }
 
 
     addHook('beforeEach', python`    return conf`)
     addHook('beforeEach', python`\n`)
 
-    if (mobile) setUpMobileNative(test, addHook)
-    else {
-        if (emulator) setUpWithEmulators(test, addHook)
-        else setUpBrowsers(test, addHook)
+    if (mobile) {
+        if(test.env.app) {
+            addHook('beforeEach', python`@pytest.fixture(scope="function")`)
+            addHook('beforeEach', python`def app():`)
+            addHook('beforeEach', python`    return ${test.env.app}\n`)
+        }
+        if(test.env.browser) {
+            addHook('beforeEach', python`@pytest.fixture(scope="function")`)
+            addHook('beforeEach', python`def browser_name():`)
+            addHook('beforeEach', python`    return ${test.env.browser}\n`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
            addHook("beforeEach", python`    return ${test.env.browser}\n`)
```